saleor/order/emails.py

```diff
@@ -73,6 +107,20 @@ def send_order_confirmation(order_pk, user_pk=None):
     )
 
 
+@app.task
+def send_staff_order_confirmation(order_pk):
+    """Send order confirmation email."""
+    staff_email_data = collect_staff_data_for_email(
+        order_pk, STAFF_CONFIRM_ORDER_TEMPLATE
+    )
+    send_templated_mail(**staff_email_data)
+    events.email_sent_event(
+        order=staff_email_data["context"]["order"],
+        user=None,
+        email_type=events.OrderEventsEmails.ORDER,
```

Comment:
I'm not sure we should create an event about it?

Reply:
you're right, 
fied